support/docker/dev/Dockerfile

```diff
@@ -15,10 +15,22 @@ WORKDIR /home/user/PeerTube
 # Configure Cloud9 IDE to use PeerTube's source directory as workspace (-w).
 RUN sudo sed -i "s/-w \/home\/user/-w \/home\/user\/PeerTube/" /etc/supervisord.conf
 
+# Install dependencies.
+RUN yarn install
+
 # Configure Janitor for PeerTube.
 ADD janitor.json /home/user/
 RUN sudo chown user:user /home/user/janitor.json
 
 # Configure and build PeerTube.
-RUN yarn install \
- && npm run build
```

Comment:
Nit: Why do you no longer pre-build PeerTube?

Reply:
`npm run dev` will build the client on the fly

Reply:
What @Chocobozzz said. Closing.